src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Wasm/PackedSimd.cs

```diff
@@ -14,7 +14,7 @@ namespace System.Runtime.Intrinsics.Wasm
     [CLSCompliant(false)]
     public abstract class PackedSimd
     {
-        public static bool IsSupported { [Intrinsic] get { return false; } }
+        public static bool IsSupported { [Intrinsic] get { return IsSupported; } }
```

Comment:
Could you please add a comment why it's done this way?
I was looking at this yesterday and it made no sense... :-)

Would be nice to have comments in the other places as well, at least eventually.

Reply:
There are quite a lot of intrinsics, I'll do that in a follow-up PR.

Reply:
`[Intrinsic]` means that the code you see may not be what is actually executed. The self-recursive implementation is used a lot with `[Intrinsic]` methods.